src/rdb.c

```diff
@@ -3952,3 +3952,41 @@ rdbSaveInfo *rdbPopulateSaveInfo(rdbSaveInfo *rsi) {
     }
     return NULL;
 }
+
+/* Restore the replication ID / offset from the RDB file
+ * return 1 if rdbSaveInfo is valid */
```

Comment:
"valid" here means that the save info contained replication ID and offset and could be successfully restored, right? Let's make the comment more explicit?

```suggestion
 * return 1 if replication ID and offset were restored from the rdbSaveInfo */
```